### PR TITLE
Make nav highlight work for child/inner pages

### PR DESCRIPTION
**Pull request by huttj**
Small change to make the nav highlight work on _all_ pages.

It might need revisiting, but it works well on the new patient, patient details, and report details pages, right now.

### CODE REVIEW

**Comment by huttj** [comment]
Hm..I didn't touch the patients controller...weird.

File: app/views/layouts/application.html.slim [hunk 1]
```diff
@@ -83,14 +83,20 @@ html
...
     javascript:
-      var url = window.location;
-      // Will only work if string in href matches with location
-      $('ul.nav a[href="'+ url +'"]').parent().addClass('active');
 
-      // Will also work for relative and absolute hrefs
-      $('ul.nav a').filter(function() {
-          return this.href == url;
-          }).parent().addClass('active');
+      // Todo: This should be cleaned up
```
Comment (awong-dev) [inline app/views/layouts/application.html.slim]: in the future, please use TODO(username): format. That way we more easily know who to ask clarification from.

**Comment by awong-dev** [comment]
@emilyville Can you take a look at this? The jQuery bits are something I'm not super confident in

File: app/views/layouts/application.html.slim (hunk 1 above)
Comment (huttj) [inline app/views/layouts/application.html.slim]: Ah, that's more of a "who wrote this" than a "who should do this"

That makes sense. Thanks. :smile:

**Comment by awong-dev** [comment]
Preemptively merging for demo.

**Pull request merged**